scripts/main-built.js

```diff
@@ -154,23 +154,24 @@ function refreshLegislatorFilters() {
  * Applies approriate table formatting for the current Browser size.
  */
 function formatTableForBrowserSize() {
-  // if the table is displayed
-  if($('#search-results').css('display') != 'none'){
-    // get whether this is for mobile or not
-    if($(window).width() <= SMALL_BREAKPOINT){
-      $('.district-cell, .election-cell').hide();
-      $('.results-cell > p:nth-child(2)').show();
-      $('button.results-cell').html('\>');
-    } else if(SMALL_BREAKPOINT < $(window).width() && $(window).width() <= MID_BREAKPOINT){
-      $('#score-header').html('CC SCORE');
-      $('#election-header').html('LAST PRES. RESULT');
-      $('.district-cell, .election-cell').show();
-      $('.results-cell > p:nth-child(2)').hide();
-      $('button.results-cell').html('TAKE ACTION');
-    } else {
-      $('#score-header').html('CLIMATE CABINET SCORE');
-      $('#election-header').html('LAST PRESIDENTIAL RESULT');
-    }
+  // apply small breakpoint changes
+  if($(window).width() <= SMALL_BREAKPOINT){
```

Comment:
oh man, we really should be building things with a JS framework and a style system that handles breakpoints for us

Reply:
very much agree